### ISSUE TITLE
Is this a complete demo?

### ISSUE THREAD

**Issue by mike-2020** [FeiGeChuanShu/ncnn_Android_PP-TinyPose#3]
Thanks for your nice work.

But want to double confirm that if this is a complete demo...

Look into source code, it only implemented TinyPose model. But to make TinyPose work efficently, do I need to add a person detection model in front of TinyPose?

**Comment by FeiGeChuanShu** [comment]
> Thanks for your nice work.
> 
> But want to double confirm that if this is a complete demo...
> 
> Look into source code, it only implemented TinyPose model. But to make TinyPose work efficently, do I need to add a person detection model in front of TinyPose?

yes, you should add a person detector for multi person condition
